File: Python/well_of_ideas.py
def well(x):
	good = 0
	for case in x:
		if case == 'good':
			good += 1
	if good > 2:
		return 'I smell a series!'
	elif good >= 1:
		return 'Publish!'
	else:
		return 'Fail!'

File: JavaScript/well_of_ideas.js
function well(array) {
	let count = 0;
	for (let i = 0; i < array.length; i++) {
		if (array[i] === 'good') {
			count++
		}
	}

	let output = count >= 1 ? 'Publish!' : 'Fail!';
	return count > 2 ? 'I smell a series!' : count >= 1 ? 'Publish!' : 'Fail!';
}